Packs/CheckPointHarmonyEndpoint/Integrations/CheckPointHarmonyEndpoint/CheckPointHarmonyEndpoint.py

```diff
@@ -0,0 +1,2703 @@
+            "harmony-ep-ioc-list": ioc_list_command,
+            "harmony-ep-ioc-update": ioc_update_command,
+            "harmony-ep-ioc-create": ioc_create_command,
+            "harmony-ep-ioc-delete": ioc_delete_command,
+            "harmony-ep-policy-rule-assignments-get": rule_assignments_get_command,
+            "harmony-ep-policy-rule-assignments-add": rule_assignments_add_command,
+            "harmony-ep-policy-rule-assignments-remove": rule_assignments_remove_command,
+            "harmony-ep-policy-rule-install": rule_policy_install_command,
+            "harmony-ep-policy-rule-modifications-get": rule_modifications_get_command,
+            "harmony-ep-policy-rule-metadata-list": rule_metadata_list_command,
+            "harmony-ep-push-operation-status-list": push_operation_status_list_command,
+            "harmony-ep-push-operation-get": push_operation_get_command,
+            "harmony-ep-push-operation-abort": push_operation_abort_command,
+            "harmony-ep-anti-malware-scan": anti_malware_scan_command,
+            "harmony-ep-anti-malware-update": anti_malware_update_command,
+            "harmony-ep-anti-malware-restore": anti_malware_restore_command,
+            "harmony-ep-forensics-indicator-analyze": indicator_analyze_command,
+            "harmony-ep-forensics-file-quarantine": file_quarantine_command,
+            "harmony-ep-forensics-file-restore": file_restore_command,
+            "harmony-ep-remediation-computer-isolate": remediation_computer_isolate_command,
+            "harmony-ep-remediation-computer-deisolate": remediation_computer_deisolate_command,
+            "harmony-ep-computer-list": computer_list_command,
+            "harmony-ep-agent-computer-restart": computer_reset_command,
+            "harmony-ep-agent-computer-shutdown": computer_shutdown_command,
+            "harmony-ep-agent-computer-repair": computer_repair_command,
+            "harmony-ep-agent-process-information-get": process_information_get_command,
+            "harmony-ep-agent-process-terminate": process_terminate_command,
+            "harmony-ep-agent-registry-key-add": agent_registry_key_add_command,
+            "harmony-ep-agent-registry-key-delete": agent_registry_key_delete_command,
+            "harmony-ep-agent-file-copy": agent_file_copy_command,
+            "harmony-ep-agent-file-move": agent_file_move_command,
+            "harmony-ep-agent-file-delete": agent_file_delete_command,
+            "harmony-ep-agent-vpn-site-add": agent_vpn_site_add_command,
+            "harmony-ep-agent-vpn-site-remove": agent_vpn_site_remove_command,
+        }
+
+        if command == "test-module":
+            return_results(test_module(client))
+        elif command in commands:
+            return_results(commands[command](args, client))
+        else:
+            raise NotImplementedError(f"{command} command is not implemented.")
+
+    except Exception as e:
+        clear_integration_context()
```

Comment:
Why?

Reply:
I want to clear the integration context in case an exception occurs in the scheduled command and the integration context is not clear yet.